addons/website_event_track/views/event_track_templates_agenda.xml

```diff
@@ -6,14 +6,27 @@
     <t t-call="website_event.layout">
         <!-- No tracks -->
         <div t-if="not tracks_by_days" class="container">
-            <div class="h2 mb-3">No track found.</div>
-            <div t-if="search_key" class="alert alert-info text-center">
-                <p class="m-0">We did not find any track matching your <strong t-out="search_key"/> search.</p>
-            </div>
-            <div t-else="" class="alert alert-info text-center" groups="event.group_event_user">
-                <a target="_blank" t-att-href="'/web?#action=website_event_track.action_event_track_from_event&amp;active_id=%s' % event.id">
-                    <p class="m-0">Schedule some tracks to get started</p>
-                </a>
+            <div class="col-12">
+                <div class="d-flex flex-column align-items-center justify-content-center text-center">
+                    <div class="m-3 o_wevent_exhibitor_svg">
+                        <t t-call="website_event.event_svg"/>
+                    </div>
+                    <h2 class="mb-2">No track found.</h2>
+                    <p>
```

Comment:
You could save a bunch of lines and one indentation level applying the condition in the `<p>` tag directly (in the other template too).
```xml

<p t-if="search_key">We couldn't find any track matching your <strong t-out="search_key"/> search.</p>
<p t-else="">We couldn't find any track at this moment.</p>

```